### ISSUE TITLE
Set up "dev" branch and tag properly

### ISSUE THREAD

**Issue by franz-josef-kaiser** [EasyPost/easypost-node#37]
While it's nice to have a `version` field in the `package.json`, it's not sufficient in most cases. Please: tag properly and make releases, there's even a dedicated page for it.

Easy and brief tutorial:

```sh
# List the last 10 commits with their short hash as one-liner:
$ git log --oneline --max-count=10

# Take a hash and add a new tag
$ git tag -a 1a2b3c4 -m "Hey! This is our newest Nodejs library version!"

# Publish time!
$ git push && git push --tags

# Done! Easy, ain't it?
```

And please offer a `dev` branch so we can have a merge target and something that we can use for testing before running with all the new things into a production environment.

```sh
# Add a new branch
$ git checkout -b dev

# Make changes or merge here… 
# Then merge the branch into dev with a new commit – so it's easy to revert (just in case)
$ git merge issue-42-new-feature dev --no-ff

# Now push to the remote named "origin" and track it
$ git push -u origin dev
```

Thanks in advance!

**Comment by ajacksified** [comment]
We've recently grown our front-end / node team and are working on improvements to the SDK; in the meantime, it's best to rely on NPM rather than git for node module versions. Thanks for your feedback.

**Comment by franz-josef-kaiser** [comment]
@ajacksified You honestly have no resources to add the minimum of what makes version control what it is – and explain in the same sentence that you have grown your team? What I am asking for is work of 1-2h … top. If I could send PRs that contain just tags, I would, but this is not possible, so I ask another time: Could you please reopen this issue and get a single resource off of their day-to-day task and tag properly? Thank you.

**Comment by ajacksified** [comment]
@franz-josef-kaiser - [we've released v3.0.0-rc.1](https://www.easypost.com/blog/2017-01-05-node-sdk-v3.html). It is up on NPM as 3.0.0-rc.1, and tagged here as well.
